backend/scripts/application_stop.sh

```
@@ -1,58 +1,51 @@
 #!/bin/bash
 export PATH="/usr/bin:/bin:$PATH"
 
-echo "=== [APPLICATION_STOP] 커피빵 게임 서버 종료 ==="
+echo "=== [APPLICATION_STOP] 커피빵 게임 서버 강제 종료 ==="
 
 cd /opt/coffee-shout
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)